src/seedu/addressbook/data/AddressBook.java

```diff
@@ -21,16 +22,17 @@
  *  - The tags in each person point to tag objects in the master list. (== equality)
  */
 public class AddressBook {
-
+	private ArrayList<Tagging> allTaggingList;
```

Comment:
Indentation problem here? Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.